plugin.json

```diff
@@ -0,0 +1,36 @@
+{
+  "id": "marketplace-addon",
+  "name": "Marketplace Addon",
+  "description": "Automatically update installed plugins to latest versions",
+  "version": "0.1.0",
+  "min_server_version": "5.14.0",
+  "server": {
+    "executables": {
+        "linux-amd64": "server/dist/plugin-linux-amd64",
+        "darwin-amd64": "server/dist/plugin-darwin-amd64",
+        "windows-amd64": "server/dist/plugin-windows-amd64.exe"
+    }
+  },
+  "settings_schema": {
+    "settings": [{
+      "key": "NotificationChannelName",
+      "display_name": "Notification Channel",
+      "help_text": "A channel name to send notifications related to plugin updates.",
+      "type": "text"
+    },{
+      "key": "MarketplaceAPIAddress",
+      "display_name": "Marketplace API's Address",
+      "type": "text",
+      "help_text": "Marketplace API used to fetch latest plugins from the Marketplace.",
+      "placeholder": "https://api.integrations.mattermost.com",
+      "default": "https://api.integrations.mattermost.com"
```

Comment:
I am not sure if we are going to need it here. I think we should use the url defined in the server config because if at runtime the config for url is changed our plugin should fetch plugin from the newer url.

Reply:
We might also need to clarify the case when plugin marketplace is not enabled on the server or it might change in between so what kind of impact it should have on the plugin.

Reply:
@lieut-data What's your take on above questions/suggestions?